.github/workflows/node_modules-publish.yml

```diff
@@ -11,6 +11,8 @@ jobs:
     timeout-minutes: 5
     env:
       GIT_DIR: git-dir.git
+      HEAD_SHA: ${{ github.event.workflow_run.pull_requests[0].head.sha }}
```

Comment:
Ugh, I've never seen this to be a list. `github.event.workflow_run.head_sha` seemed quite fine? But then again, maybe `pull_requests[]` is just what I was looking for for *reliably* getting the original information of the triggering `pull_request`. This would make a lot of things simpler, or even possible again (see https://github.com/cockpit-project/cockpit-machines/pull/143). So I'll definitively look at this again.